package.json

```
@@ -3,5 +3,16 @@
     "@commitlint/cli": "^12.1.4",
     "@commitlint/config-conventional": "^12.1.4",
     "husky": "^7.0.1"
+  },
+  "scripts": {
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.php": "php ./vendor/bin/phpcs --standard=PSR12"
```

Da es jetzt ein config file für phpcs gibt, muss der standard nicht mehr als argument angegeben werden.

```suggestion
    "*.php": "php ./vendor/bin/phpcs"
```